apps/trading/components/select-market/select-market.tsx

```diff
@@ -170,7 +170,7 @@ export const SelectMarketPopover = ({
           </div>
         ) : (
           <table className="relative text-sm w-full whitespace-nowrap">
-            {pubKey && (positions?.length ?? 0) > 0 ? (
```

Comment:
non blocking: no need to use nullish coalescing operator, null is also falsy
`pubKey && positions?.length && markets?.length`